src/FakeItEasy/IHideObjectMembers.cs

```diff
@@ -6,9 +6,7 @@ namespace FakeItEasy
 
     /// <summary>
     /// Hides standard Object members to make fluent interfaces
-    /// easier to read. Found in the source of Autofac: <see cref="!:http://code.google.com/p/autofac/"/>
-    /// Based on blog post here:
-    /// <see cref="!:http://www.clariusconsulting.net/blogs/kzu/archive/2008/03/10/58301.aspx"/>

```

Comment:
Link is broken, and I couldn't find another article explaining this.

Reply:
http://blogs.clariusconsulting.net/kzu/how-to-hide-system-object-members-from-your-interfaces/ ?

Reply:
Doh! Your Google-fu is better than mine. Still no HTTPS, though. I'll amend.

Reply:
> Your Google-fu is better than mine

Today, perhaps.